src/utils/cx.ts

```diff
@@ -0,0 +1,5 @@
+export const cx = (...args: (string | undefined | boolean)[]) =>
```

Comment:
Should have a better naming here. What is the purpose of this function?

Reply:
This function to combine string when we want to combine style of css.